packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2973,13 +2976,13 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
 
   /// An integer that is guaranteed to be greater than the parent's, if any.
   /// The element at the root of the tree must have a depth greater than 0.
-  int get depth => _depth;
-  int _depth;
+  int? get depth => _depth;
```

Comment:
can we make this non-null for consistency with AbstractNode?

Reply:
Done. I've done some refactoring to avoid testing if depth is null. Changes have been done in https://github.com/flutter/flutter/pull/64672/commits/c0e5d54a217727487197e04802ada46c3cb882ab.

Reply:
Seems reasonable!